### DISCUSSION TITLE
Excited to try on my PC -- sad it appears to not support Mac OS

### DISCUSSION THREAD

**Discussion by lopperman** [GCuser99/SeleniumVBA#109] [General]
I work on both Mac and PC platforms -- though I do lean towards Mac for most of my day to day.
This is really impressive and although I wish more open source VBA was compatible with both platforms, I think as long as someone has access to a PC, the benefit of this testing library will serve it's purpose.

One small suggestion -- may be change the first line in the documentation from:

`A comprehensive Selenium wrapper for browser automation developed for MS Office VBA`

to something like: 

`A comprehensive Selenium wrapper for browser automation developed for MS Office VBA running on Windows`

**Comment by GCuser99** [comment]
Thanks for the feedback @lopperman - I made the change that you suggested.

Yeah, I hear you about Mac compatibility. I suspect that redesigning for cross-platform would be a significant effort, plus I don't have a Mac to do the development work. Maybe someone in the community can take that on eventually - you are certainly welcome to fork a try! 

Take care,

Mike
